Mathlib/Data/Real/EReal.lean

```diff
@@ -800,6 +800,19 @@ theorem top_add_coe (x : ℝ) : (⊤ : EReal) + x = ⊤ :=
   rfl
 #align ereal.top_add_coe EReal.top_add_coe
 
+/--For any extended real number `x` which is not `⊥`, the sum of `⊤` and `x` is equal to `⊤`.-/
+@[simp]
+theorem top_add_ne_bot {x : EReal} (h : x ≠ ⊥) : ⊤ + x = ⊤ := by
```

Comment:
```suggestion
theorem top_add_of_ne_bot {x : EReal} (h : x ≠ ⊥) : ⊤ + x = ⊤ := by
```
and the other one `add_top_of_ne_bot`. 
Hypotheses should come after `_of_`, and not be "inlined".

Reply:
Done. Thanks!